### ISSUE TITLE
Not able to complete jailbreak process on beta iOS 15.4 (19E5209h) iPhone XS

### ISSUE THREAD

**Issue by jurgenvisser** [opa334/Dopamine#562]
Hi I'm trying to jailbreak my iPhone XS on beta iOS 15.4 (19E5209h) and i cannot seen to get a completed jailbreak via the Dopamine app.

The app is installed through TrollStore as posted on the website and I've tried Turing off and on: WiFi, Airplane mode and tweak injection as if read some about that online. 

It gets stuck on the "Initializing Jailbreak Environment" step after choosing Sileo and Zebra, or just Sileo.

After turing on "Verbose Logs" I cannot find a log file but if i record my screes and look at the lost step it tried to to i see this:
> [injectDylibViaRop] Preparation done, now injecting!
> [injectDylibViaRop] dlopen: 0x1BBD4D118, dlerror: 0x1BBD4EAE8
> [arbCall] Set thread state for arbitrary call
> [arbCall] Started thread, waiting for it to finish...

Those are the last lines after is reboots the device and be in an unjailbroken state in stead of respringing the device and showing Sileo and Zebra installed.

I can't find anything else to guide me thought this. Please let me know how to fix this. Would love to use this jailbreak!

**Comment by opa334** [comment]
Does this produce a panic log? (Settings -> Privacy -> Analytics -> Analytics and Data) If so, send it.

**Comment by jurgenvisser** [comment]
https://drive.google.com/file/d/1tL-g_dyAP__vo_Vgv8g7_7jSyRaDdy5_/view?usp=drivesdk

Hope this will work for you. Couldn’t find any other way to include it via
GitHub directly. If this doesn’t work please let me k is it WeTransfer does
work for you!

Hope to get some insight in what I’m doing wrong soon!

Kind Regards,


J.V.

**Comment by jurgenvisser** [comment]
https://drive.google.com/file/d/1tL-g_dyAP__vo_Vgv8g7_7jSyRaDdy5_/view?usp=drivesdk

I also replied via email. Hope that this helps. Please let me know if my problem has a fix!

**Comment by opa334** [comment]
You need to make the file public...

**Comment by jurgenvisser** [comment]
Done.

**Comment by opa334** [comment]
The only thing this panic tells me is that there is something going horribly wrong

**Comment by jurgenvisser** [comment]
Yeah I have a lot of panic logs. To me they look all the same. But I have
no idea why this didn’t work. I did a Xina jailbreak just fine. So I don’t
know.

Can I provide anything else to you to help solve this?

**Comment by hostedbyjustus** [comment]
@jurgenvisser Is there any update on this issue? After these years, the situation might've changed. Otherwise this issue may be marked as solved.
